firmware/system.h

```diff
@@ -12,9 +12,9 @@
 extern "C" {
 #endif
 
-// Needed for delay intrinsics
-#define _XTAL_FREQ 16000000
-
+//16 MHz osc, but normally through PLL (96 MHz) divided by 2
```

Comment:
`48 MHz* osc`? Is the actual PIC running at 16MHz, but the delay intrinsics require knowledge of the PLL, or is the whole thing running at 48MHz?

Reply:
I believe the whole thing is at 48 MHz. I'm less clear on the theory, but I'm fairly certain the measured value is correct. I'd suggest merging in since this fixes a fairly major bug, and we can sort out comments later if they need updating

Reply:
Fine with me, I approve and we can merge!